indico/modules/events/registration/models/forms.py

```diff
@@ -396,8 +396,7 @@ def active_registrations(self):
 
     @property
     def sender_address(self):
-        contact_email = self.event.contact_emails[0] if self.event.contact_emails else None
-        return self.notification_sender_address or contact_email
+        return self.notification_sender_address
```

Comment:
The property is used in exactly one place - let's get rid of it and use `notification_sender_address` there directly.